src/components/hoverable-visuals.js

```diff
@@ -56,6 +56,7 @@ AFRAME.registerComponent("hoverable-visuals", {
 
     let interactorOne, interactorTwo;
     const interaction = AFRAME.scenes[0].systems.interaction;
+    if (!interaction.options.leftHand.entity) return; //DOMContentReady workaround
```

Comment:
since this is reaching pretty far into interaction system, i'd suggest maybe exposing this as a simple `ready` flag?

Reply:
(only partially convinced that's a better idea, but wanted to surface)